modules/data.atmosphere/R/download_noaa_gefs_efi.R

```diff
@@ -0,0 +1,111 @@
+#' download_NOAA_GEFS_EFI
+#'
+#' @param start_date start date for met forecast
+#' @param sitename NEON site name
+#' @param outfolder filepath to save ensemble member .nc files
```

Comment:
```suggestion
#' @param sitename NEON site name
#' @param outfolder filepath to save ensemble member .nc files
#' @param start_date start date for met forecast
```

(I think Roxygen will resort the params in the compiled docs to match the function signature, but when reading here it helps to keep them in order)